addons/web/views/webclient_templates.xml

```diff
@@ -745,8 +745,26 @@
             <t t-set="head_web">
                 <script type="text/javascript">
                     odoo.session_info = <t t-raw="session_info"/>;
+                    /**
+                     * The menu data contains the definition of the action that is needed to load the webclient.
+                     * We are loading the menu here, before the js assets are loaded to ensure that they will be ready
+                     * faster, and that the session of the webclient needs to wait less long for the menu data.
+                     */
```

Comment:
As discussed I'd also get rid of this comment entirely, it's quite verbose and the commit message should be enough if someone wants to really understand the rationale.